backend/printcmm.ml

```diff
@@ -287,7 +295,14 @@ let operation d = function
 let rec expr ppf = function
   | Cconst_int (n, _dbg) -> fprintf ppf "%i" n
   | Cconst_natint (n, _dbg) -> fprintf ppf "%s" (Nativeint.to_string n)
-  | Cconst_vec128 ({ low; high }, _dbg) -> fprintf ppf "%016Lx:%016Lx" high low
+  | Cconst_vec128 ({ word0; word1 }, _dbg) ->
+    fprintf ppf "%016Lx:%016Lx" word0 word1
+  | Cconst_vec256 ({ word0; word1; word2; word3 }, _dbg) ->
+    fprintf ppf "%016Lx:%016Lx:%016Lx:%016Lx" word0 word1 word2 word3
+  | Cconst_vec512
+      ({ word0; word1; word2; word3; word4; word5; word6; word7 }, _dbg) ->
+    fprintf ppf "%016Lx:%016Lx:%016Lx:%016Lx:%016Lx:%016Lx:%016Lx:%016Lx" word0
+      word1 word2 word3 word4 word5 word6 word7
```

Comment:
It feels like the reverse order would
be more natural.

Reply:
Flipped